ur_description/urdf/ur.ros2_control.xacro

```diff
@@ -6,10 +6,18 @@
     initial_positions:=${dict(shoulder_pan_joint=0.0,shoulder_lift_joint=-1.57,elbow_joint=0.0,wrist_1_joint=-1.57,wrist_2_joint=0.0,wrist_3_joint=0.0)}
     script_filename output_recipe_filename
     input_recipe_filename tf_prefix
-    hash_kinematics robot_ip">
+    hash_kinematics robot_ip
+    sim_gazebo:=false
+    sim_ignition:=false">
```

Comment:
I changed it.
i also added a commit where i removed the build testing files, because I think these are the reasons which caused all the errors. Can you please check it again?